### PR TITLE
Jump2logicalPage

### PR DESCRIPTION
**Pull request by frank-ulrich-weber**

### CODE REVIEW

**Comment by sebastian-meyer** [comment]
Could you please describe what this feature is supposed to do? :)

**Comment by frank-ulrich-weber** [comment]
Sure! Sorry! :-) I don't skip writing an issue the next time.
The land registry offices need a functionality to enter the logical page number. If they do a search, they know the document and the logical page number. The search results in exactly (approximately 90% :-) one search result. They don't want to do a click on the document within the list view and again for the page within the page view. They would like to do a search and as a reault the page view opens imediately on the needed logical page.

File: dlf/plugins/search/class.tx_dlf_search.php [hunk 1]
```diff
@@ -654,17 +679,24 @@ public function main($content, $conf) {
...
-				$linkConf['additionalParams'] = \TYPO3\CMS\Core\Utility\GeneralUtility::implodeArrayForUrl($this->prefixId,
-					array (
-						'id' => $results->current()['uid'],
-						'highlight_word' => preg_replace('/\s\s+/', ';', $results->metadata['searchString']),
-						'page' => count($results[0]['subparts']) == 1?$results[0]['subparts'][0]['page']:1
-					), '', TRUE, FALSE);
+				$additionalParams['id'] = $results->current()['uid'];
+				$additionalParams['highlight_word'] = $results->current()['uid'];
+				$additionalParams['page'] = $results->current()['uid'];
+
```
Comment (sebastian-meyer) [inline dlf/plugins/search/class.tx_dlf_search.php]: This seems to be wrong. All of the parameters are set to `uid`?

**Review by sebastian-meyer — CHANGES_REQUESTED** [review]
Please check my comment below.

**Review by sebastian-meyer — APPROVED** [review]

**Pull request merged**